Update _index.md (#27609)
DataDog/documentation: content/en/cloud_cost_management/datadog_costs/_index.md

{{< callout url="https://www.datadoghq.com/product-preview/daily-datadog-costs/" btn_hidden="false" header="Join the Preview!">}}
Daily Datadog costs in Cloud Cost Management are in Preview. If you're interested in this feature, complete the form to request access.

> <p>Customers purchasing Datadog through a Reseller Partner or an External Marketplace non-drawdown contract are currently ineligible for the Preview.</p>

Reviewer: Hi @katherinebroner, would it be possible to turn this into a more positive statement so that it lists the types of customers who would be eligible. For example users on an Enterprise plan, etc.? I don't know what the inverse of "Reseller Partner" or "External Marketplace non-drawdown contact" would be.

Author: hi @rtrieu yes good flag - i think this could work: 

"Customers with a direct contract through Datadog or an External Marketplace drawdown contract are eligible for the Preview."

> <p>Customers purchasing Datadog through a Reseller Partner or an External Marketplace non-drawdown contract are currently ineligible for the Preview.</p>

Reviewer: ```suggestion
<p>Only customers with a direct contract through Datadog or an External Marketplace drawdown contract are eligible for the Preview.</p>
```

Author: @rtrieu this looks good, thank you!
